packages/styling/src/styles/PulsingBeaconAnimationStyles.ts

```diff
@@ -0,0 +1,90 @@
+import { IRawStyle, keyframes } from '@uifabric/merge-styles/lib/index';
+
+const DEFAULT_DURATION = '14s';
+const DEFAULT_DELAY = '2s';
+const DEFAULT_ITERATION_COUNT = '1';
+
+function _continuousPulseStepOne(beaconColorOne: string, innerDimension: string): IRawStyle {
```

Comment:
I'm pretty sure you can have numbers in function names, it just can't start with a number.